strawberryfields/compilers/xcov.py

```diff
@@ -122,8 +122,17 @@ def compile(self, seq, registers):
             # determine the modes that are acted on by the symplectic transformation
             used_modes = [x.ind for x in seq[0].reg]
 
+            # Since this compiler does not allow for displacements
+            # when its parameter are passed to the GaussianUnitary compiler
+            # the latter either returns a GaussianTransform + MeasureFock
+            # or just MeasureFock. This is because the GaussianUnitary checks
+            # if the symplectic matrix is just the identity it simply elides it
```

Comment:
```suggestion
            # or just MeasureFock. This is because the GaussianUnitary checks
            # if the symplectic matrix is just the identity; if so, it simply elides it
```

Reply:
Also, I didn't know of the word 'elides'! I learnt something